rita_exit/src/network_endpoints/mod.rs

```diff
@@ -99,7 +99,9 @@ fn decrypt_exit_client_id(
         }
     };
 
-    let decrypted_id: ExitClientIdentity = match serde_json::from_str(&decrypted_string) {
+    //let old_decrypted_id: OldExitClientIdentity = serde_json::from_str(&decrypted_string);
+    //let decrypted_id: ExitClientIdentity = serde_json::from_str(&decrypted_string);
```

Comment:
don't leave lines commented, that's what git history is for.

Reply:
whoops forgot to remove these, fixed